Loop54.Shared/Model/Request/GetEntitiesByAttributeRequest.cs

```diff
@@ -38,6 +40,32 @@ public GetEntitiesByAttributeRequest(string attributeName, string[] attributeVal
         /// </summary>
         public EntityCollectionParameters ResultsOptions { get; set; } = new EntityCollectionParameters();
 
+        /// <summary>
+        /// Provides human-readable labels in the Portal.
+        /// </summary>
+        public RequestAliasData RequestAlias { get; set; }
+
+        /// <summary>
+        /// Provides human-readable labels in the Portal.
+        /// </summary>
+        public class RequestAliasData
+        {
+            /// <summary>
+            /// Specify an alias for this attribute name.
+            /// </summary>
+            public string Name { get; set; }
+
+            /// <summary>
+            /// Specify an alias for this attribute value.
+            /// </summary>
+            public string Value { get; set; }
+
+            /// <summary>
+            /// Specify a more detailed description for this attribute name�value pair. Will be shown in the Portal.
```

Comment:
This is some kind of a weird Unicode dash in "name�value" that is rendered as ? on GitHub.